style.css

```
@@ -0,0 +1,186 @@
+/* basic grid-layout, sets grid for head, body and footer */
+.grid-layout-body {
+  display: grid;
+  height: 100vh;
+  grid-template-columns: 1fr 280px 1fr; /*1fr 280px 1fr;*/
+  grid-template-rows: auto auto 80px;
+}
+
+.grid-item-header {
+  grid-column: 2 / 3;
+  grid-row: 1 / 2;
+}
+
+.grid-item-main {
+  grid-column: 2 / 3;
+  grid-row: 2 / 3;
+}
+
+.grid-item-footer {
+  grid-column: 2 / 3;
+  grid-row: 3 / 4;
+}
+/*Basic style for body, Head, main, footer*/
+body {
+  font-family: "Nunito Sans", sans-serif;
+  font-size: 14px;
+  background-color: #e6a937;
+  box-sizing: border-box;
+}
+header {
+  background-color: rgb(166, 213, 219);
+  margin-bottom: 5px;
+  padding: 10px;
+}
+
+main {
+  background-color: deeppink;
+  margin: 5px 0px;
```

If you using 0px you can shorten this with only 0
```suggestion
  margin: 5px 0;
```